thunderfish/csvmaker.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from .dataloader import load_data
+from .consistentfishes import consistent_fishes
+from .harmonicgroups import harmonic_groups
+from .powerspectrum import multi_resolution_psd
+
+
+def extract_main_freqs_and_db(data, samplerate):
```

Comment:
Why do you need it? For testing freqs_and_db_csv() you only need to supply a list of frequencies and powers. This function does not care about whether these lists have been computed by harmonic_groups() or supplyed manually, like
```
freqs = [200, 350, 432]
pow = [0.87, 0.34, 7.2]
freqs_and_db_csv('test.csv', freqs, pow)
```